R/tm_t_pp_basic_info.R

```diff
@@ -1,12 +1,15 @@
-#' Template: Basic Info
+#' Template: Patient Profile Basic Info
 #'
-#' Creates a basic info template.
+#' Creates a valid expression to generate a patient profile basic info report using ADaM datasets.
 #'
 #' @inheritParams template_arguments
-#' @param vars (`character`)\cr variable names to be shown in Basic Info tab.
-#' @param patient_id (`character`)\cr patient ID.
-#' @keywords internal
+#' @param vars (`character`)\cr names of the variables to be shown in the "Basic Info" tab.
```

Comment:
```suggestion
#' @param vars (`character`)\cr names of the variables to be shown in the `Basic Info` tab.
```

Is the formatting better with such quotes?

Reply:
I have updated the wording here to "... shown in the table." which I think makes more sense.

Reply:
cool